src/pages/lib/auth/delete_user/q/platform/[platform].mdx

```diff
@@ -1,23 +1,99 @@
 export const meta = {
-  title: `Delete user`,
-  description: `Delete a user`
+  title: `Delete user account`,
+  description: `Learn how to delete a user account.`
 };
 
-import ios0 from '/src/fragments/lib/auth/native_common/delete_user/common.mdx';
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{ ios: ios0 }} />
+Empowering users to delete their account can improve trust and transparency. You can programmatically enable self-service account deletion with Amplify Auth. Users can delete their own account through your app interface by confirming intent before final removal. In this guide we will review how you can enable this for your users.
```

Comment:
recommendation:

Empowering users to delete their account can improve trust and transparency. You can programmatically enable self-service account deletion with Amplify Auth. In this guide, we will review how you can enable this feature for your users.